src/pages/developer-portal.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+import {
+  SEO,
+  InterstitialTitle,
+  StickyFooter,
+  ResponsiveSpacer,
+} from 'components';
+// eslint-disable-next-line import/named
+import { Testimonial, Hero, FooterCTA, features } from 'components/home';
+
+const SEO_TITLE = 'Backstage Developer Portal';
```

Comment:
@oblongcoffee all the titles and main copy of the page are gathered here in these strings. If you want to make other pages, simply copy/paste this one within the `src/pages` directory and change these strings.